src/components/GameAbout/index.tsx

```diff
@@ -3,73 +3,126 @@ import React, { useEffect, useRef, useState } from 'react'
 import classNames from 'classnames'
 
 import * as Images from '@/assets/images'
+import Button from '@/components/Button'
 
+import Sticker from '../Sticker'
 import styles from './GameAbout.module.scss'
 
 export interface GameAboutProps {
+  titleSmall?: string
+  titleLarge?: string
   description: string
   gameName?: string
+  sticker?: {
+    label: string
+    withIcon?: React.ReactNode
+  }[]
+  buttonLink?: {
+    expanded?: boolean
+    onClick?: () => void
+  }
   classnames?: {
     container?: string
-    title?: string
-    description?: string
+    titleSmall?: string
+    titleLarge?: string
     gameTitle?: string
+    description?: string
+    buttonLink?: string
   }
 }
 
-const GameAbout = ({ description, gameName, classnames }: GameAboutProps) => {
+const GameAbout = ({
+  titleSmall,
+  titleLarge,
+  description,
+  gameName,
+  sticker,
+  classnames,
+  buttonLink
+}: GameAboutProps) => {
   const aboutText = useRef<HTMLParagraphElement | null>(null)
   const [isClamped, setIsClamped] = useState(false)
-  const [expanded, setExpanded] = useState(false)
+  const [expanded, setExpanded] = useState(buttonLink?.expanded || false)
 
   useEffect(() => {
     if (aboutText && aboutText.current) {
-      const { clientHeight, scrollHeight } = aboutText.current
-      console.log({ clientHeight, scrollHeight })
-      setIsClamped(scrollHeight > clientHeight)
+      const element = aboutText.current
+      const hasOverflow = element.scrollHeight > element.clientHeight
+      setIsClamped(hasOverflow || expanded)
+    }
+  }, [description, expanded])
+
+  const handleShowMore = () => {
+    setExpanded(!expanded)
+    if (buttonLink?.onClick) {
+      buttonLink.onClick()
     }
-  }, [])
+  }
 
   return (
-    <div className={classnames?.container}>
+    <div className={classNames(styles.container, classnames?.container)}>
+      {titleSmall && (
+        <div className={classNames('title-sm', classnames?.titleSmall)}>
+          {titleSmall}
+        </div>
+      )}
+      {titleLarge && (
+        <div className={classNames('title', classnames?.titleLarge)}>
+          {titleLarge}
+        </div>
```

Comment:
Do we need both?

Reply:
We should use ternaries instead of `&&` too https://www.crocoder.dev/blog/react-conditional-rendering/

Reply:
We need both, the small one is applied for the "About" we have in the old live design, which also has the game name using the same large title so it would look odd. The second is being used in the "About This Game" in the new design.